### DISCUSSION TITLE
Add `text-pretty` and `text-stable` utility.

### DISCUSSION THREAD

**Discussion by jguddas** [tailwindlabs/tailwindcss#11646] [Ideas]
`text-wrap: pretty` is super new and just rolling out in chrome.

Is there a general approach when utilities like this get added?

https://groups.google.com/a/chromium.org/g/blink-dev/c/rwBWqqOB_ag/m/ea9dZXvaDQAJ?pli=1
https://drafts.csswg.org/css-text-4/#valdef-text-wrap-pretty

**Comment by mybearworld** [comment]
It's currently not supported in any browsers

**Reply by jguddas** [reply]
Yes very unsupported, you can try it out using https://www.google.com/chrome/dev.

Do you know if we have an x% adoption minimum before we add new utilities?
